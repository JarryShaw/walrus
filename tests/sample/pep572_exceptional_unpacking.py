data = [15, 16, 17]
print(info := (13, 14, *data))
print(info)
