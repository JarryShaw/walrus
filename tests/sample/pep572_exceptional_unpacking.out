(13, 14, 15, 16, 17)
(13, 14, 15, 16, 17)
